api/models.py

```diff
@@ -187,8 +187,4 @@ class Meta:
         verbose_name = "Security User"
 
     def save(self, *args, **kwargs):
```

Comment:
do we still need this?

Reply:
I left it cause I guess you mentioned this `After discussions with the android team it seems like we'll not be implementing use of SMSs for now`

Reply:
yeah.. so i think we can get rid of the `save` methods since we don't have any more customizations.

Reply:
Alright.

Reply:
Done.